### DISCUSSION TITLE
Error: Hydration failed because the initial UI does not match what was rendered on the server.

### DISCUSSION THREAD

**Discussion by zenotsai066** [vercel/next.js#50256] [Help] [answered]
### Summary

I initialized the project using ```npx create-next-app@latest``` and wrote a very simple structure in src/page.tsx. I don't know why there is an error
 ```Error: Hydration failed because the initial UI does not match what was rendered on the server.``` after executing ```yarn run dev```.

I just need to replace ```<section>``` with ```<div>``` and it works fine, which confuses me a lot...

![image](https://github.com/vercel/next.js/assets/66027308/d794bc6a-eaf8-4c24-9e41-ed7a8977ef1c)

### Example
```jsx
// Error
export default function Home() {
  return (<section>
        <p>123</p>
      </section>
  )
}
```

```jsx
//  work
export default function Home() {
  return <div>
    <p>123</p>
  </div>
}
```
_No response_

**Comment by icyJoseph** [comment, marked as answer]
I can't reproduce. Is this in pages or app dir? Do you have any extensions that mutate the DOM?

**Reply by zenotsai066** [reply]
@icyJoseph   app dir
![image](https://github.com/vercel/next.js/assets/66027308/eae25ab5-bdab-485f-a987-e22a59a7003a)

**Reply by zenotsai066** [reply]
It may be caused by a certain extension. After switching to another browser, it works normally. Can you tell me how to troubleshoot this type of error?

**Reply by icyJoseph** [reply]
Right, incognito mode usually works fine to discard extensions.

And well, how to find out which one it is... turn off one by one... 😿

**Reply by zenotsai066** [reply]
@icyJoseph Thank you very much. I compared the differences between the HTML returned by the network and the HTML rendered by the browser, and solved this problem!!

**Reply by icyJoseph** [reply]
Which extension was it, if you don't mind sharing?

**Reply by silkquasar** [reply]
> Which extension was it, if you don't mind sharing?

For me in Brave, Chrome and Firefox the problem is caused by this extension:

Startpage Privacy Protection 1.1.5
